### ISSUE TITLE
force_ssl was removed from REST API Shop

### ISSUE THREAD

**Issue by nick-friedrich** [phpclassic/php-shopify#244]
I get this error in shopify partners dashboard:￼

force_ssl was removed from REST API Shop

**Comment by hjarnaa** [comment]
I get the same error, do we have a fix for this?

**Comment by anasiqbalkhan** [comment]
Hi, any update on this ?

**Comment by dvdheiden** [comment]
The Shopify warning is a bit confusing, but already tells you what to do:

> Update deadline: Apr 1, 2022 - Update to version 2021-07 or later

Unless your using this attribute yourself, the warning probably is about using an older API version. This package is recently updated to use the newer API version. When you update the package to the latest version, the packages uses a newer version of the API, see: https://github.com/phpclassic/php-shopify/commit/32ec82af1bad7c1cf52363b4711fcc3a820abfbc

That probably solves your problem.

**Comment by hjarnaa** [comment]
Thanks a billion @dvdheiden I have updated the Package, and so far it seems to work.

**Comment by milkbottlelough** [comment]
Yep, me too.  Updating has fixed this issue for me also (`composer update phpclassic/php-shopify`)

thanks to @dvdheiden for the update suggestion; usually the very last thing I remember to do and usually the only thing that fixes my issues 😅
